includes/Canvas.h

```diff
@@ -1,85 +1,155 @@
 #pragma once
 
 #include "Color.h"
-#include "abstract/AbstractWindow.h"
-#include <SDL2/SDL.h>
+#include "Renderer.h"
+#include <cmath>
 #include <string>
 #include <vector>
 
+namespace doengine
+{
+
+#ifdef TODO
```

Comment:
Ummm. Idk, but this looks awkard to me. I think it would be better to do this:
```suggestion
#warning "TODO: Implement, refactor or delete Canvas"
#ifdef 0
```